kegtab/build.gradle

```diff
@@ -97,11 +95,11 @@ android {
 dependencies {
     implementation 'com.hoho.android:usbserial:0.3.0-rc1@aar'
     implementation 'com.google.android.gms:play-services-gcm:9.4.0'
-    implementation files('libs/kegbot-api/kegbot-api.jar')
+    api files('libs/kegbot-api/kegbot-api.jar')
     implementation 'com.android.support:support-v13:26.1.0'
     implementation 'com.squareup.okhttp:okhttp:2.0.0'
     implementation 'com.squareup:otto:1.3.4'
-    implementation 'com.google.guava:guava:24.1-jre'
+    implementation 'com.google.guava:guava:20.0'
```

Comment:
Turns out this was actually the cause of the issue I was seeing.  I tested each available version starting at `18.0` and saw the crash when I included version `21.0`.  I also noticed that starting with version `21.1`, there is no longer just a vanilla version of the package and instead you have to choose between `jre` and an `android` version.